spec/generators/install_generator_spec.rb

```diff
@@ -1,27 +1,33 @@
-# -*- encoding: utf-8 -*-
```

Comment:
Well this PR is intended for Slices v2, so cleaning up any unnecessary ones still has value. But yes, once we are ruby >= 2.0 we can delete them all without worrying about anything.

Btw re. dropping support for ruby 1.9.3 - I thought that rails 4 was ruby 2.0 only, but I was wrong. So in theory we could continue support for 1.9.3 in Slices v3 if we wanted.